jobbergate-core/pyproject.toml

```diff
@@ -0,0 +1,79 @@
+[tool.poetry]
+name = "jobbergate-core"
+version = "3.4.3"
+description = "Jobbergate Core"
+authors = ["Omnivector Solutions <info@omnivector.solutions>"]
+license = "MIT"
+readme = "README.rst"
+repository = "https://github.com/omnivector-solutions/jobbergate"
+packages = [ { include = "jobbergate_core" } ]
```

Comment:
Including a `packages` section is only necessary if the package name isn't the same only with dashes replaced by underscores. In this case it wouldn't be needed.

Reply:
solved in [8328c22](https://github.com/omnivector-solutions/jobbergate/pull/228/commits/8328c2296dee7b226a46536345949df3752182db)